modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -0,0 +1,693 @@
+== kn
+
+The `kn` task uses the `kn` command line utility to complete operations on Knative resources, such as services, revisions, or routes.
+
+.Example usage of the `kn` task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+ name: kn-run
+spec:
+ pipelineSpec:
+   tasks:
+   - name: kn-run
+     taskRef:
+       resolver: cluster
+       params:
+       - name: kind
+         value: task
+       - name: name
+         value: kn
+       - name: namespace
+         value: openshift-pipelines
+     params:
+     - name: ARGS
+       value: [version]
+----
+
+.Supported parameters for the `kn` task
+[options="header"]
+|===
+| Parameter | Description | Type | Default value
+|`ARGS` |The arguments for the `kn` utility. |`array` |`- help`
+|===
+
+.Changes from the `kn` `ClusterTask`
+
+* The `KN_IMAGE` parameter was removed.
+
+[discrete]
+[id="op-taskref-kn-apply_{context}"]
+== kn-apply
+
+The `kn-apply` task deploys a specified image to a Knative Service. This task uses the `kn service apply` command to create or update the specified Knative service.
+
+.Example usage of the `kn-apply` task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+ name: kn-apply-run
+spec:
+ pipelineSpec:
+   tasks:
+   - name: kn-apply-run
+     taskRef:
+       resolver: cluster
+       params:
+       - name: kind
+         value: task
+       - name: name
+         value: kn-apply
+       - name: namespace
+         value: openshift-pipelines
+     params:
+     - name: SERVICE
+       value: "hello"
+     - name: IMAGE
+       value: "gcr.io/knative-samples/helloworld-go:latest"
+----
+
+.Supported parameters for the `kn-apply` task
+[options="header"]
+|===
+| Parameter | Description | Type | Default value
+|`SERVICE` |The Knative service name |`string` |
+|`IMAGE` |The fully qualified name of the image to deploy |`string` |
+|===
+
+.Changes from the `kn-apply` `ClusterTask`
+
+* The `KN_IMAGE` parameter was removed.
+
+
+[discrete]
+[id="op-taskref-maven_{context}"]
+== maven
+
+The `maven` task runs a Maven build.
+
+.Example usage of the `maven` task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: build-and-deploy
+spec:
+# ...
+  tasks:
+# ...
+  - name: build-from-source
+    taskRef:
+      resolver: cluster
+      params:
+      - name: kind
+        value: task
+      - name: name
+        value: maven
+      - name: namespace
+        value: openshift-pipelines
+    workspaces:
+    - name: source
+      workspace: shared-workspace
+# ...
+----
+
+.Supported parameters for the `maven` task
+[options="header"]
+|===
+| Parameter | Description | Type | Default value
+|`GOALS` | The Maven goals to run |`array` |`- package`
+|`MAVEN_MIRROR_URL` |The Maven repository mirror url |`string` |
+|`SUBDIRECTORY` |The subdirectory within the `source` workspace on which the task runs the Maven build. |`string` |`.`
```

Comment:
Per ISG you don't have to use "on which" - you can end in a preposition to avoid an awkward construction

```suggestion
|`SUBDIRECTORY` |The subdirectory within the `source` workspace that the task runs the Maven build on. |`string` |`.`
```